bases/renku_data_services/background_jobs/main.py

```diff
@@ -20,6 +20,7 @@ async def short_period_sync() -> None:
     await bootstrap_user_namespaces(config)
     await config.syncer.events_sync(config.kc_api)
     await sync_admins_from_keycloak(config.kc_api, Authz(config.authz_config))
+    await fix_mismatched_project_namespace_ids(config)
```

Comment:
question: does this need to be done regularly if it's just a one-time fix?

Reply:
It is just a one time fix.

See https://github.com/SwissDataScienceCenter/renku-data-services/issues/278 on how we can address the problem of constantly re-running such tasks.